plugin/commentary.vim

```diff
@@ -44,6 +44,12 @@ function! s:go(...) abort
     endif
   endfor
 
+  if exists('b:commentary_startofline') && b:commentary_startofline == 1
```

Comment:
Use `if get(b:, 'commentary_startofline')` rather comparing to the explicit value of `1`.

Reply:
ok